helm/scylla-operator/values.yaml

```diff
@@ -49,6 +52,9 @@ serviceAccount:
 ## SecurityContext holds pod-level security attributes and common container settings.
 securityContext: {}
 
+# Replicas for Webhook Server
```

Comment:
Although it's not surprising, please explicitly add a warning comment that setting replicas to a lower value then 2 means it won't be HA and creations or updates of Scylla CRs will fail during operator upgrades or disruptions.

Reply:
Thanks for your feedback! I added a comment and added a check before creating pdbs